ui-modules/blueprint-composer/app/components/catalog-selector/catalog-selector.directive.js

```diff
@@ -243,4 +285,6 @@ function controller($scope, $element, $q, $uibModal, $log, $templateCache, palet
 
     // allow downstream to configure this controller and/or scope
     (composerOverrides.configurePaletteController || function() {})(this, $scope, $element);
+    
+    $scope.paletteItemClasses = "col-xs-3";
```

Comment:
This doesn't seems to be used so can be removed

Reply:
yes, removed in #94